Client/Coordinators/Library/LibraryCoordinator.swift

```diff
@@ -32,10 +36,38 @@ class LibraryCoordinator: BaseCoordinator, LibraryPanelDelegate {
     func start(with homepanelSection: Route.HomepanelSection) {
         libraryViewController.setupOpenPanel(panelType: homepanelSection.libraryPanel)
         libraryViewController.delegate = self
+        libraryViewController.navigationHandler = self
         libraryViewController.resetHistoryPanelPagination()
     }
 
-    // MARK: - LibraryCoordinatorDelegate
+    // MARK: - LibraryNavigationHandler
+
+    func start(panelType: LibraryPanelType, navigationController: UINavigationController) {
+        switch panelType {
+        case .bookmarks:
+            createBookmarksCoordinator(navigationController: navigationController)
+        case .history:
+            break
+        case .downloads:
+            break
+        case .readingList:
+            break
```

Comment:
Can we add some comments with the tasks those will be done in? Like `// History coordinator will be implemented in #12345`. It's a bit easier to keep track this way

Reply:
yes sure